app/controllers/newflow/login_controller.rb

```diff
@@ -18,7 +18,6 @@ def login
         success: lambda {
           clear_signup_state
           user = @handler_result.outputs.user
-          Sentry.set_user(id: user.id) if Rails.env.production?
```

Comment:
Removing this because nothing here throws? Or some other reason?

Reply:
[just moved, not removed](https://github.com/openstax/accounts/pull/1293/files#diff-766c34fd6533171eaf54300c153f89d6002c35c02cfc9c5b219251f85180ad07R39)